Automattic/wp-calypso
# A4A: add referrals guide content

Related to https://github.com/Automattic/jetpack-genesis/issues/355

## Proposed Changes

Adding content to the Referral Modal. Images are stored locally within Calypso, while videos are in wp cdn.
Note: there's a slight "jump" between steps we are aware of. It will be fixed in the future.

https://github.com/Automattic/wp-calypso/assets/60262784/3c61187c-ae09-4201-8c81-b4731e9e2977



## Why are these changes being made?



## Testing Instructions



- Using live link below navigate to any `/marketplace` link and open referral modal using `i` button on the Referral toggle in the top right corner.
- Check all steps of the Modal

## Pre-merge Checklist
